test/Microsoft.AspNetCore.Mvc.Razor.Test/TagHelpers/TagHelperComponentTagHelperTest.cs

```diff
@@ -288,5 +334,21 @@ public Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
                 return Task.CompletedTask;
             }
         }
+
+        private class TestAddTagHelperComponent : ITagHelperComponent
+        {
+            public int Order => 0;
+
+            public void Init(TagHelperContext context)
+            {
+                context.Items["Key2"] = "Value2";
```

Comment:
What is this `Items` entry used for? Same for the similar code in later test components.